src/app/CommandHandler.cpp

```diff
@@ -132,14 +130,19 @@ CHIP_ERROR CommandHandler::ProcessCommandDataElement(CommandDataElement::Parser
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR CommandHandler::AddStatusCode(const CommandPathParams & aCommandPathParams,
+CHIP_ERROR CommandHandler::AddStatusCode(const ConcreteCommandPath & aCommandPath,
                                          const Protocols::SecureChannel::GeneralStatusCode aGeneralCode,
                                          const Protocols::Id aProtocolId, const Protocols::InteractionModel::Status aStatus)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
     StatusElement::Builder statusElementBuilder;
 
-    err = PrepareCommand(aCommandPathParams, true /* isStatus */);
+    chip::app::CommandPathParams commandPathParams = { aCommandPath.mEndpointId,
+                                                       0, // GroupId
+                                                       aCommandPath.mClusterId, aCommandPath.mCommandId,
+                                                       (chip::app::CommandPathFlags::kEndpointIdValid) };
```

Comment:
nit: are the parentheses needed?

Reply:
They are not needed; just copied this from other places where we set up this struct and did not think about it enough.